graph-manager-docs/source/slack-integration.md

```diff
@@ -3,68 +3,75 @@ title: Integrating Graph Manager with Slack
 sidebar_title: Slack
 ---
 
-Apollo's Slack integration brings your server's performance metrics and analytics data from Apollo Graph Manager directly to your team's Slack workspace so you can be notified of potential issues proactively. The integration does two main things:
+Apollo Graph Manager can send notifications about your data graph to your team's Slack workspace. This enables you to act on any potential issues as soon as they arise.
```

Comment:
Restricting this to "potential issues" here doesn't make as much sense with the addition of change notifications. How about something a little bit more generic?
```suggestion
Apollo Graph Manager can send notifications about your data graph to your team's Slack workspace. This enables you to stay up to date on changes to your schema and your graph's performance as soon as they happen.
```